spec/features/global_announcements_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Global announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    describe 'no global announcements' do
+      before do
+        instance.announcements.clear
```

Comment:
You're asking about clearing the database. It's two different things were talking about here.
If it can be factored out, sure. But clearing the database at the start of all tests is not the solution for writing state preconditions.